### ISSUE TITLE
improve descriptions of concepts in Georeference Verification Status Vocabulary

### ISSUE THREAD

**Issue by AaronWilton** [hiscom/hispid#113]
all terms in this vocabulary have concept title as the description.
The are pretty self evident, but should we expand along the following lines?

verification required:  verification of the georeference  is required
verified:  the georeference has been verified 
verified by collector:  the georeference has been verified by the collector
verified by custodian:  the georeference has been verified by the custodian of the specimen
verified by contributor:  the georeference was verified by the agent who contributed the specimen 
unverifiable:  the georeference is unable to be verified

**Comment by ben3000** [comment]
The closest match in our system to this element has the following terms. You'll note that we don't really care who verified it, but what the outcome was:

- (nil): Locality has not been queried or validated before. Coordinates and identity are accepted at face value.
- REV: Under Review. Coordinates have been queried, either as a result of an exception report from me using BIOCLIM analysis or from a FloraBase user query or the like.
- POS: Position Changed. Coordinates have been checked and found to be inconsistent or incorrect. Coordinates have been subsequently changed.
- IDE: Identity Changed. Coordinates were correct. However, the identity of the specimen was incorrect and changed.
- PCN: Position Confirmed. Coordinates were correct. However, the identity of the specimen cannot be confirmed at this stage.
- PIC: Position And Identity Confirmed. Coordinates have been checked and confirmed to be correct. Specimen has been re-identified and was correct. No changes were necessary.
- UNV: Unverifiable. Both position and identity were found to be correct on the basis of the label data. However, a specialist has deemed the locality to be inconsistent with current knowledge of the taxon.

I don't think we should adopt the terms exactly, as they are too short and thus not descriptive enough. I just wanted to add this to show that we could take a slightly different angle on what we report.

**Comment by AaronWilton** [comment]
yes, I remember that we discussed the difference between a verification status and a verification outcome.  I think the decision/discussion was that we did care who had done the verification (may impact on reliability), and that the outcome/changes would be reflected in the data.  
Also the status of the identification is separated here too.

At least that's my recollection...

**Comment by acvaughan** [comment]
Leaving aside the ensuing discussion between Ben and Aaron, I think the expanded descriptions are a good idea.

**Comment by nielsklazenga** [comment]
HISCOM 2016: agreed
